test/Dialect/Torch/decompose-complex-ops.mlir

```diff
@@ -494,3 +494,26 @@ func @torch.aten.batch_norm(%arg0: !torch.vtensor<[?,?,?,?],f32>, %arg1: !torch.
   %0 = torch.aten.batch_norm %arg0, %arg1, %arg2, %arg3, %arg4, %false, %float1.000000e-01, %float1.000000e-05, %false : !torch.vtensor<[?,?,?,?],f32>, !torch.vtensor<[?],f32>, !torch.vtensor<[?],f32>, !torch.vtensor<[?],f32>, !torch.vtensor<[?],f32>, !torch.bool, !torch.float, !torch.float, !torch.bool -> !torch.vtensor<[?,?,?,?],f32>
   return %0 : !torch.vtensor<[?,?,?,?],f32>
 }
+
+// -----
+// CHECK-LABEL: func @hardswish(
```

Comment:
Pls fix the format. Did you use `mlir/utils/generate-test-checks.py`? The generated code should have the correct format already.

Reply:
Done.